src/xdocs/config_metrics.xml

```diff
@@ -101,6 +101,22 @@
         <source>
 &lt;module name=&quot;BooleanExpressionComplexity&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <div class="wrapper">
+          <pre>
```

Comment:
xdocs should have `source` tag not, `div` or `pre`.

Reply:
![Screenshot from 2020-03-03 04-09-44](https://user-images.githubusercontent.com/45921807/75724770-1367a280-5d05-11ea-8503-e265001af899.png)

Reply:
done